spec/models/group_spec.rb

```diff
@@ -82,4 +83,13 @@
       expect(group.name).to include(leader.name)
     end
   end
+
+  describe '#requests' do
+    it 'returns an array of users who have requested to join' do
+      group = FactoryGirl.create(:open_group)
+      user = FactoryGirl.create(:student, intent: 'on_campus', draw: group.draw)
+      Membership.create(group: group, user: user, status: 'requested')
```

Comment:
Lol, see my [previous comment](https://github.com/YaleSTC/vesta/pull/121/files#r98221781) - we weren't even consistent last night 😛.